packages/destination-actions/src/destinations/blackbaud-raisers-edge-nxt/index.ts

```diff
@@ -19,23 +24,27 @@ const destination: DestinationDefinition<Settings> = {
     },
     refreshAccessToken: async (request, { auth }) => {
       // Return a request that refreshes the access_token if the API supports it
-      const res = await request('https://oauth2.sky.blackbaud.com/token', {
+      const res = await request<RefreshTokenResponse>(SKY_OAUTH2_TOKEN_URL, {
         method: 'POST',
+        headers: {
```

Comment:
Did you find some reason you need to set `Content-Type` explicitly? This should be the default value.

Reply:
Updated